### DISCUSSION TITLE
When my screen refresh rate is 360Hz, lazer caps me at 60fps.

### DISCUSSION THREAD

**Discussion by AwesomeRedDragon** [ppy/osu#29178] [Q&A]
Not sure how to fix this, or if it's a hardware thing on my end, but I'm using a Razer Blade 15 (I know, I know, playing on laptop) and it lets me set the refresh rate of my screen, either to 60hz or 360hz. Naturally, I prefer 360hz, but for whatever reason lazer does not like that, and whenever my screen is set to 360hz the in-game fps is locked to 60, on all frame limiter settings.

https://github.com/user-attachments/assets/40c70b55-5bfe-46fc-9e61-7c287cebdc0d

Here's a video of me switching between 60hz and 360hz, where you can see that the fps is lowered to 60 as I switch to 360hz.

My logs are here, although I'm not sure they help since this might be a hardware thing.
[compressed-logs.zip](https://github.com/user-attachments/files/16406252/compressed-logs.zip)

If anyone knows what might be the issue, please let me know!

**Comment by peppy** [comment]
If you restart the game does this resolve?

**Reply by AwesomeRedDragon** [reply]
Unfortunately not, I even uninstalled both osu and osu lazer entirely (reinstalled later) to see if that fixed it but no luck.
